src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py

```diff
@@ -124,7 +124,7 @@ def get_metadata(self) -> dict:
         metadata["Ecephys"].update(NeuroScopeRecordingInterface.get_ecephys_metadata(xml_file_path=xml_file_path))
         session_start_time = get_session_start_time(str(xml_file_path))
         if session_start_time is not None:
-            metadata = dict_deep_update(metadata, dict(NWBFile=dict(session_start_time=session_start_time)))
+            metadata["NWBFile"]["session_start_time"] = session_start_time
```

Comment:
This type of integration is great though, really showcases how much nicer the new class is to work with